flexbox/src/androidTest/java/com/google/android/flexbox/test/FlexboxAndroidTest.kt

```diff
@@ -0,0 +1,3843 @@
+
+        val textView3 = activity.findViewById<TextView>(R.id.text3)
+        assertThat(textView3.right, `is`(flexboxLayout.right - flexboxLayout.paddingRight))
+    }
+
+    @Test
+    @FlakyTest
+    @Throws(Throwable::class)
+    fun testAlignContent_center() {
+        val activity = activityRule.activity
+        val flexboxLayout = createFlexboxLayout(R.layout.activity_align_content_test,
+                object : Configuration {
+                    override fun apply(flexboxLayout: FlexboxLayout) {
+                        flexboxLayout.alignContent = AlignContent.CENTER
+                    }
+                })
+
+        assertThat(flexboxLayout.alignContent, `is`(AlignContent.CENTER))
+
+        onView(withId(R.id.text1)).check(isLeftAlignedWith(withId(R.id.flexbox_layout)))
+        onView(withId(R.id.text2)).check(isCompletelyRightOf(withId(R.id.text1)))
+        onView(withId(R.id.text3)).check(isCompletelyBelow(withId(R.id.text1)))
+
+        val textView1 = activity.findViewById<TextView>(R.id.text1)
+        val textView3 = activity.findViewById<TextView>(R.id.text3)
+        var spaceAboveAndBottom = flexboxLayout.height - textView1.height - textView3.height
+        spaceAboveAndBottom /= 2
+
+        assertThat(textView1.top, isEqualAllowingError(spaceAboveAndBottom))
+        assertThat(flexboxLayout.bottom - textView3.bottom, isEqualAllowingError(spaceAboveAndBottom))
+        assertThat(flexboxLayout.flexLines.size, `is`(2))
+    }
+
+    @Test
+    @FlakyTest
+    @Throws(Throwable::class)
+    fun testAlignContent_spaceBetween() {
+        val flexboxLayout = createFlexboxLayout(R.layout.activity_align_content_test,
+                object : Configuration {
+                    override fun apply(flexboxLayout: FlexboxLayout) {
+                        flexboxLayout.alignContent = AlignContent.SPACE_BETWEEN
+                    }
+                })
+
+        assertThat(flexboxLayout.alignContent, `is`(AlignContent.SPACE_BETWEEN))
+        onView(withId(R.id.text1)).check(isLeftAlignedWith(withId(R.id.flexbox_layout)))
+        onView(withId(R.id.text1)).check(isTopAlignedWith(withId(R.id.flexbox_layout)))
+        onView(withId(R.id.text2)).check(isCompletelyRightOf(withId(R.id.text1)))
+        onView(withId(R.id.text2)).check(isTopAlignedWith(withId(R.id.flexbox_layout)))
+        onView(withId(R.id.text3)).check(isBottomAlignedWith(withId(R.id.flexbox_layout)))
+        onView(withId(R.id.text3)).check(isLeftAlignedWith(withId(R.id.flexbox_layout)))
+        assertThat(flexboxLayout.flexLines.size, `is`(2))
+    }
+
+    @Test
+    @FlakyTest
+    @Throws(Throwable::class)
+    fun testAlignContent_spaceBetween_withPadding() {
+        val flexboxLayout = createFlexboxLayout(R.layout.activity_align_content_test,
+                object : Configuration {
+                    override fun apply(flexboxLayout: FlexboxLayout) {
+                        flexboxLayout.alignContent = AlignContent.SPACE_BETWEEN
+   
```

Comment:
This expression is not added to the widthSumFirstRow variable. Please remove the line break